addon/routes/external.js

```diff
@@ -24,14 +25,17 @@ export default class ExternalRoute extends Route {
   };
 
   @service router;
-  @service fastboot;
   @service intl;
 
   constructor() {
     super(...arguments);
     this.templateName = 'external';
   }
 
+  get fastboot() {
```

Comment:
I think it would be good to add a comment here why we're using a getter instead of `@service`, to avoid the next person working on this to move this to a service again